KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/DropdownToolbar.java

```diff
@@ -198,6 +210,16 @@ public void propertyChange(PropertyChangeEvent evt) {
                             disableSearch = true;
                         }
                         
+                        //set the data source list
+                        try {
+                            //SleuthkitCase skDb = ((Case)evt.getNewValue()).getSleuthkitCase();
+                            dataSources = getDataSourceList();
+                        } catch (TskCoreException ex) {
+                            logger.log(Level.SEVERE, "Error getting text index info", ex); //NON-NLS
+                            disableSearch = true;
+                        } catch (NoCurrentCaseException ex) {
+                            Exceptions.printStackTrace(ex);
```

Comment:
Don't print a stack trace. It might make sense to do the same disable search as the TskCoreException above